spec/resty/resolver_spec.lua

```diff
@@ -163,6 +163,14 @@ describe('resty.resolver', function()
     before_each(function()
       tmpname = io.tmpfile()
 
+      tmpname:write('# nameserver updated  in comentary\n')
```

Comment:
How about calling `write` just once? I think it looks better:

```lua
      tmpname:write(
        [[# nameserver updated  in comentary\n
          #nameserver updated  in comentary\n
          #comentary nameserver\n
          ..............]])
```

(Not sure if the `\n` needs to be there)

Reply:
done in d49b712